docs/reference/index-config.md

```diff
@@ -361,6 +361,10 @@ In plain language:
 - it should only contain latin letter `[a-zA-Z]` digits `[0-9]` or (`.`, `-`, `_`).
 - the first character needs to be a letter.
 
+:::caution
+For field names containing the `.` character, you will need to escape it when referencing them. Otherwise the `.` character will be interpreted as a JSON object property access. Because of this, it is recommended to avoid using field names containing the `.` character.
```

Comment:
```suggestion
For field names containing the `.` character, you will need to escape it when referencing them in queries. Otherwise the `.` character will be interpreted as a JSON object property access. Because of this, it is recommended to avoid using field names containing the `.` character.
```